plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -142,27 +144,62 @@ func (c *podSecurityPolicyPlugin) Admit(a admission.Attributes) error {
 		return nil
 	}
 
+	// sort by name to make order deterministic
+	// TODO: add priority field to allow admins to bucket differently
```

Comment:
open issue or something for this?  Sorting on name leads people do weird things.

Reply:
It's already here: https://github.com/kubernetes/kubernetes/pull/45358